aspnetcore/security/authentication/sociallogins.md

```diff
@@ -143,28 +124,31 @@ Once you enter your Facebook credentials, then you will be redirected back to th
 
 You are now logged in using your Facebook credentials.
```

Comment:
This is from a bad merge with original branch. Main topic for social logins should not have anything Facebook-specific - thanks for catching, merge highlighting confused me and I missed it.